maxim/logger/anthropic/client.py

```
@@ -1,18 +1,18 @@
-from anthropic import Anthropic
-
 from ..logger import (
     Logger,
 )
+from anthropic import Anthropic
 from .message import MaximAnthropicMessages
 
 
 class MaximAnthropicClient:
+
     def __init__(self, client: Anthropic, logger: Logger):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)